cinema/serializers.py

```diff
@@ -1 +1,66 @@
-# write serializers here
+from rest_framework import serializers
+
+from .models import CinemaHall, Genre, Actor, Movie, MovieSession
+
+
+class CinemaHallSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = CinemaHall
+        fields = ["id", "name", "rows", "seats_in_row", "capacity"]
+
+
+class GenreSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Genre
+        fields = ["id", "name"]
+
+
+class ActorSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Actor
+        fields = ["id", "first_name", "last_name", "full_name"]
```

Comment:
The `full_name` property in the `ActorSerializer` is included in the fields, but it is a property method in the `Actor` model. Ensure that `full_name` is correctly defined as a property in the model and that it can be serialized. If necessary, you can use a `SerializerMethodField` to handle this property.